amundsen_application/static/js/components/TableDetail/WatermarkLabel/styles.scss

```diff
@@ -8,9 +10,24 @@
     height: 40px;
     margin-right: 8px;
     width: 12px;
+    margin-top: 2px;
```

Comment:
Mind not having this margin?

Reply:
It is important to have it properly aligned, you don't agree?

Reply:
I guess our eyeballs just have a different perspective on what "alignment" looks like. But I can see how you find this helps better align. I'm fine with this.